nix/sources.json

```diff
@@ -28,12 +28,12 @@
         "branch": "cronos",
         "description": "A CosmosSDK application for moving assets on and off of EVM based, POW chains",
         "homepage": "",
-        "owner": "crypto-org-chain",
+        "owner": "yihuang",
         "repo": "gravity-bridge",
-        "rev": "630e7235f4b2345177b571813ab316f01cdbfb93",
-        "sha256": "021n33b05x41bc65b42af179q12sx55dsh4si9xc1rjjvbl3dpd5",
+        "rev": "68fb5b5f8ce596f7380e51991bc62472f00e5cd4",
+        "sha256": "00sss25r07vsm23m92xamh44m5c8yj1m4ivbv925hkf366bi6dnc",
         "type": "tarball",
-        "url": "https://github.com/crypto-org-chain/gravity-bridge/archive/630e7235f4b2345177b571813ab316f01cdbfb93.tar.gz",
+        "url": "https://github.com/yihuang/gravity-bridge/archive/68fb5b5f8ce596f7380e51991bc62472f00e5cd4.tar.gz",
```

Comment:
Don't merge before this PR is merged: https://github.com/crypto-org-chain/gravity-bridge/pull/15

Reply:
done